source/docs/zero-to-robot/step-3/radio-programming.rst

```diff
@@ -2,198 +2,112 @@
+.. note:: Newer versions of the radio firmware will have the version listed at the bottom.  If you do not see the version listed you have firmware prior to 1.2.0.
 
-## Application Notes
+1. On the Vivid-Hosting [firmware releases](https://frc-radio.vivid-hosting.net/miscellaneous/firmware-releases) page download the proper firmware for the current firmware version you have.  Always choose the `Radio Variant`.
 
-By default, the Radio Configuration Utility will program the radio to enforce the 4Mbps bandwidth limit on traffic exiting the radio over the wireless interface. In the home configuration (AP mode) this is a total, not a per client limit. This means that streaming video to multiple clients is not recommended.
+2. Copy the SHA-256 key below the firmware you downloaded.
 
-The Utility has been tested on Windows 7, 8 and 10. It may work on other operating systems, but has not been tested.
+3. Paste that key into the :guilabel:`Checksum` box of the :guilabel:`Firmware Upload` section at the bottom of the configuration page we navigated to above.
 
-### Programmed Configuration
+4. Click :guilabel:`Browse...` and select the firmware file you downloaded.
 
-.. image:: /docs/hardware/hardware-basics/images/status-lights/openmesh-radio-status-lights.png
-  :alt: Lists the names of each of the status lights on the two legal radios.
+5. Click the :guilabel:`Upload` button.
 
-The Radio Configuration Utility programs a number of configuration settings into the radio when run. These settings apply to the radio in all modes (including at events). These include:
+.. warning:: The radio will take approximately 2-3 minutes to complete firmware updates. Do not remove power during this process. Damage to the radio can occur.  When the PWR light is solid and the SYS light is slowly blinking at 1 Hz, the firmware upgrade process is complete.
 
-- Set a static IP of ``10.TE.AM.1``
-- Set an alternate IP on the wired side of ``192.168.1.1`` for future programming
-- Bridge the wired ports so they may be used interchangeably
-- The LED configuration noted in the graphic above.
-- 4Mb/s bandwidth limit on the outbound side of the wireless interface (may be disabled for home use)
-- QoS rules for internal packet prioritization (affects internal buffer and which packets to discard if bandwidth limit is reached). These rules are:
+## Robot Radio Configuration (All Teams)
 
-  - Robot Control and Status (UDP ``1110``, ``1115``, ``1150``)
-  - Robot TCP & :term:`NetworkTables` (TCP ``1735``, ``1740``)
-  - Bulk (All other traffic). (disabled if BW limit is disabled)
+.. image:: images/radio-programming/configuration-page.png
+  :alt: The top section of the radio configuration page
 
-- :term:`DHCP` server enabled. Serves out:
+This is the default configuration for the VH-109 radio to act as a Robot Radio for FRC competitions while at home. This procedure is not required when at a competition.
```

Comment:
This section is reading pretty choppy to me. Maybe something like "This section is used for configuring the VH-109 radio outside of competition. At competition, configuration will be done automatically by a provided computer and manual configuration using this page **should not be used**."

Reply:
I used your suggestion but took out the word "automatically" as I thought this may indicate they didn't need to do anything in an uninformed reading of the sentence.